src/components/organisms/pupil/OakQuizOrderQuestion/OakQuizOrder.tsx

```diff
@@ -0,0 +1,235 @@
+import React, {
+  FC,
+  createContext,
+  memo,
+  useContext,
+  useEffect,
+  useState,
+} from "react";
+import {
+  DndContext,
+  closestCenter,
+  KeyboardSensor,
+  useSensor,
+  useSensors,
+  DragEndEvent,
+  DragOverlay,
+  DragStartEvent,
+  UniqueIdentifier,
+  Announcements,
+  DndContextProps,
+  MouseSensor,
+  TouchSensor,
+} from "@dnd-kit/core";
+import {
+  arrayMove,
+  SortableContext,
+  sortableKeyboardCoordinates,
+  useSortable,
+  verticalListSortingStrategy,
+} from "@dnd-kit/sortable";
+import { CSS } from "@dnd-kit/utilities";
+import { createPortal } from "react-dom";
+
+import { OakFlex } from "@/components/atoms";
+import {
+  OakDragAndDropInstructions,
+  OakDraggable,
+  OakDroppable,
+} from "@/components/molecules";
+
+type OakQuizOrderItem = {
+  id: string;
+  label: string;
+};
+
+export type OakQuizOrderProps = {
+  /**
+   * The initial order of items
+   *
+   * this cannot be updated on subsequent renders
+   */
+  initialItems: OakQuizOrderItem[];
+  /**
+   * Notified the consumer when the order of items has changed
+   */
+  onChange?: (items: OakQuizOrderItem[]) => void;
+};
+
+const ConnectedDraggable = memo(({ id, label }: OakQuizOrderItem) => {
```

Comment:
out of interest ? Is memoization necessary to maintain the components dragged positions or for some other reason ?

Reply:
Not necessary at all anymore, in an earlier iteration I had some other gubbins in an `onDragMove` handler that was firing a lot. I'm not seeing any performance issues with this removed so i'm going to take it out 👍🏻